app/lib/proof_provider.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```

Comment:
Why `app/lib` and not `app/providers`?

Reply:
The `ActivityPub` and `OStatus` classes are likewise in `app/lib`, so I followed a precedent. Also, the namespacing is intentional to compartmentalize the proprietary vendor in such a way as to make removing it or swapping it out easier (everything is loaded more or less based on the `provider` value in `AccountIdentityProof` model)

Reply:
Yeah, seems like these should be a ProofProvider type of class, IMO, but I can definitely understand the current implementation.